Packs/Whois/Integrations/Whois/Whois.py

```diff
@@ -10,6 +10,10 @@
 
 SHOULD_ERROR = demisto.params().get('with_error', False)
 
+RATE_LIMIT_RETRY_COUNT_DEFAULT: int = 3
```

Comment:
Add another note under README -> Troubleshooting of the integration where you explain if the !ip command finishes with timeout, possible reason could be exceeding rate limit. And possible solutions will be either investigate why they reached exceeding rate limit, or set the retry to 0. This way the commands not timeout, but fail immediately. 
<img width="737" alt="image" src="https://github.com/demisto/content/assets/7270217/7412797c-7ffd-461d-a5f8-720abaab5fef">

Reply:
https://github.com/demisto/content/pull/26579/commits/3aefe389ea89693fe7dce2b5bc9ea9c1a3626605